src/App.vue

```diff
@@ -130,6 +130,14 @@
     .page-content{
       margin-left:0;
     }
+
+    .section{
+      padding: 0.75rem !important;
```

Comment:
Using `!important` may be a flaw in design. Sorry, I can check this today, but cannot it be avoided?

Reply:
Section padding is set in the variables.scss line 62 :
`$section-padding: 1.5rem 1.5rem;`

I don't know how to set it for mobile if it's not with `!important`

Reply:
Well, media queries do not change the specificity of a selector in css, so both rules, the one defined by bulma and the one you are overriding compete with equals power. In such a case, the last one wins. So the good solution would be to have the custom, app-specific css loaded after the vendor-specific css content.

Alas, there seem to be some problems with the CSS configuration:
* @cbeauchesne states somewhere in the doc that bulma css is included into app chunk instead of vendor-chunk
* if you look at the production site, the webpack plugins seem a little bit messed up, because each css chunk is included twice.

Really, there should not be the need to use `!important`, but the situation is quite a mess... @cbeauchesne  any feedbac on this?